code/clue.js

```diff
@@ -7,53 +7,267 @@ const mrGreen = {
   age: 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+  occupation: 'Entrepreneur',
+  favouriteWeapon: 'bat'
+};
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const profPlum = {
+  firstName: 'Professor Victor',
+  lastName: 'Plum',
+  color: 'plum',
+  description: 'He is an intellectual and quick wittted scholar',
+  age: 36,
+  image: 'assets/plum.png',
+  occupation: 'University Professor',
+  favouriteWeapon: 'poison'
+};
+
+const msScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'A vivacious woman who knows what she wants',
+  age: 25,
+  image: 'assets/scarlet.png',
+  occupation: 'Actress',
+  favouriteWeapon: 'rope'
+};
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'purple',
+  description: 'A grand dame who maintains her dignity',
+  age: 32,
+  image: 'assets/peacock.png',
+  occupation: 'Socialite',
+  favouriteWeapon: 'candlestick'
+};
+
+const colMustard = {
+  firstName: 'Colonel Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'A dapper colonial imperalist and dangerous military man',
+  age: 55,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel',
+  favouriteWeapon: 'dumbbell'
+};
+
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'A two-faced servant who smiles politely but sneers behind the back',
+  age: 60,
+  image: 'assets/white.png',
+  occupation: 'Housekeeper',
+  favouriteWeapon: 'knife'
+};
+
+// CREATE OBJECTS FOR ALL THE WEAPONS:
 
 const rope = {
-  name: 'rope',
-  weight: 10
-}
+  name: 'Rope',
+  weight: 5,
+  damage: 2,
+  image: 'assets/rope.png'
+};
+
+const knife = {
+  name: 'Knife',
+  weight: 3,
+  damage: 9,
+  image: 'assets/knife.png'
+};
+
+const candlestick = {
+  name: 'Candlestick',
+  weight: 8,
+  damage: 5,
+  image: 'assets/candlestick.png'
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 10,
+  damage: 8,
+  image: 'assets/dumbbell.png'
+};
+
+const poison = {
+  name: 'Poison',
+  weight: 1,
+  damage: 10,
+  image: 'assets/poison.png'
+};
+
+const axe = {
+  name: 'Axe',
+  weight: 7,
+  damage: 9,
+  image: 'assets/axe.png'
+};
+
+const bat = {
+  name: 'Bat',
+  weight: 5,
+  damage: 8,
+  image: 'assets/bat.png'
+};
+
+const trophy = {
+  name: 'Trophy',
+  weight: 7,
+  damage: 5,
+  image: 'assets/trophy.png'
+};
+
+const pistol = {
+  name: 'Pistol',
+  weight: 2,
+  damage: 9,
+  image: 'assets/pistol.png'
+};
 
 // NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 
 const suspects = [
   mrGreen,
+  profPlum,
+  msScarlet,
+  mrsPeacock,
+  colMustard,
   mrsWhite
-  // ...  and the rest
-]
+];
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
```

Comment:
Consider the benefit/disadvantage of declaring the constants first and then the array, rather than the objects directly in the array:
```javascript
const weapons = [
  {
    name: "Axe",
  ...
  }
]
```